scripts/debcheck.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+#
+# use dose-debcheck/dose-distcheck to do "repoclosure" for debian
+# Debian package dose-distcheck
+# Fedora package dose3-tools
+
+DEBIAN_RELEASE=${1:-bullseye}
+FOREMAN_VERSION=${2:-nightly}
+# user repos TBD
+
+case ${DEBIAN_RELEASE} in
+  focal)
+    # this is broken, as ubuntu needs multiple archives to be enabled to work
+    DEBIAN_PACKAGES_SOURCE=http://archive.ubuntu.com/ubuntu/dists/${DEBIAN_RELEASE}/main/binary-amd64/Packages.xz
+    ;;
+  *)
+    DEBIAN_PACKAGES_SOURCE=https://deb.debian.org/debian/dists/${DEBIAN_RELEASE}/main/binary-amd64/Packages.xz
+    ;;
+esac
+
+PUPPET_PACKAGES_SOURCE=https://apt.puppet.com/dists/${DEBIAN_RELEASE}/puppet6/binary-amd64/Packages
+FOREMAN_PACKAGES_SOURCE=https://deb.theforeman.org/dists/${DEBIAN_RELEASE}/${FOREMAN_VERSION}/binary-amd64/Packages
+PLUGINS_PACKAGES_SOURCE=https://deb.theforeman.org/dists/plugins/${FOREMAN_VERSION}/binary-amd64/Packages
+
+WORKDIR=$(mktemp -d)
+PACKAGES_DEBIAN=${WORKDIR}/Packages_debian
+PACKAGES_PUPPET=${WORKDIR}/Packages_puppet
+PACKAGES_FOREMAN=${WORKDIR}/Packages_foreman
+PACKAGES_PLUGINS=${WORKDIR}/Packages_plugins
```

Comment:
Should plugins be optional so you can also check core? Any plans to integrate this into CI pipelines?

Reply:
slow down, will ya? :)

yes, and yes, and I still need to figure out how to properly do this for ubuntu

Reply:
1. done
2. will be somewhere else ;)